recipe/conda_build_config.yaml

```diff
@@ -6,3 +6,5 @@ lmdb:
   - 0.9.22
 nodejs:
   - "18"
+folly:
+  - "2023.09.25.00"
```

Comment:
Why not pinning in the meta.yaml file, since folly is already an host dependency?

Reply:
Oh I forgot the initial pinning for `libevent-2.1.10`. I doubt there would be a global pinning of folly in conda-forge, but let's keep everything at the same place and not refactor all the branches.